development/build/index.js

```diff
@@ -117,4 +133,77 @@ function defineAllTasks() {
 
   // special build for minimal CI testing
   createTask('styles', styleTasks.prod);
+
+  // Finally, start the build process by running the entry task.
+  runTask(entryTask, { skipStats });
+}
+
+function parseArgv() {
+  const NamedArgs = {
+    BetaVersion: 'beta-version',
+    BuildType: 'build-type',
+    OmitLockdown: 'omit-lockdown',
+    SkipStats: 'skip-stats',
+  };
+
+  const entryTask = process.argv[2];
+  if (!entryTask) {
+    throw new Error('MetaMask build: No entry task specified.');
+  }
+
+  // The entry task should never be prefixed with '-' or '--', so that is easily
+  // distinguishable from other arguments.
+  if (entryTask.startsWith('-')) {
+    throw new Error(`MetaMask build: invalid entry task: ${entryTask}`);
```

Comment:
```suggestion
    throw new Error(`MetaMask build: invalid entry task: "${entryTask}"`);
```

Reply:
My practice is to double-quote substitution values in template strings, _unless_ they're the last token of the string and 
follow a colon. Is it time for me to change my ways? 🤔

Reply:
i like wrapping things so its clear if theres a trailing space or something

Reply:
I'm changing my ways. I'm never going back.

a04f202